src/components/Markdown/MarkdownEditor/MarkdownEditor.tsx

```diff
@@ -2,16 +2,11 @@ import type { FC } from 'react';
 import type { MarkdownEditorProps } from './MarkdownEditor.types';
 
 import React from 'react';
-import classNames from 'classnames/bind';
+import classNames from 'classnames';
 import MDEditor, { ICommand } from '@uiw/react-md-editor';
-import {
-  MarkdownEditorVariants,
-  MarkdownEditorModes,
-  MARKDOWN_EDITOR_TOOLBAR_TITLES
-} from './MarkdownEditor.constants';
+import { MarkdownEditorModes, MARKDOWN_EDITOR_TOOLBAR_TITLES } from './MarkdownEditor.constants';
 import styles from './MarkdownEditor.module.scss';
-
-const cx = classNames.bind(styles);
+import { Alert } from '../../Alert';
```

Comment:
nit: order of imports could be updated to be in line with zApps - although not sure we follow the structure here in zUI

Reply:
Good point!